gui/Menu.cpp

```diff
@@ -314,6 +314,8 @@ void UserInterface::KeyhandlerMenu(WORD key)
 	}
 
 	if (change) {
+		DrawMenuItems();
+		// again, this is workaround to properly re-draw menu, TODO investigate
```

Comment:
I think that shouldn't be in a codebase. I suspect it may be a problem of your distro and hw renderer to framebuffer.
This happens regularly, and the HW rendering problem is reported by many users, see: https://github.com/mborik/GPMD85Emulator/issues/21

Reply:
Yup, I'm using soft renderer too. It is weird, menu is not redrawn properly when some menu items are disabled/enabled as an response to checkbox being disabled/enabled. I was not able to fix it, found a workaround to call DrawMenuItems() two times.

Reply:
Here is a fix and correct way to handle checkboxes that changing something within current submenu: https://github.com/mborik/GPMD85Emulator/commit/9c197992e9a151556517a83d60818dd7c42e2bc1